scripts/monotone_framework_lattice.py

```diff
@@ -224,7 +226,7 @@ def main():
                       - %isLessOrEqual% the C++ function with signature `bool isLessOrEqual(const LatticeElement &LHS, const LatticeElement &RHS)`
                       - %combineValues% the C++ function with signature `bool combineValues(const LatticeElement &LHS, const LatticeElement &RHS)`
                       - %transfer% the C++ function with signature `bool transfer(TransferFunction T, const LatticeElement &RHS)`
-                      """,
+                      """,  # noqa: E501
```

Comment:
This kind of annotations are quite obscure, I think we should either disable the check in flake or add `noqa` and comment what's going on.
For this case (line too long), we might disable it. `black` should already to a good enough job in this sense.